test/test_cleaner.rb

```diff
@@ -1,82 +1,83 @@
-require 'helper'
+require "helper"
 
 class TestCleaner < JekyllUnitTest
   context "directory in keep_files" do
     setup do
       clear_dest
 
-      FileUtils.mkdir_p(dest_dir('to_keep/child_dir'))
-      FileUtils.touch(File.join(dest_dir('to_keep'), 'index.html'))
-      FileUtils.touch(File.join(dest_dir('to_keep/child_dir'), 'index.html'))
+      FileUtils.mkdir_p(dest_dir("to_keep/child_dir"))
+      FileUtils.touch(File.join(dest_dir("to_keep"), "index.html"))
+      FileUtils.touch(File.join(dest_dir("to_keep/child_dir"), "index.html"))
 
       @site = fixture_site
-      @site.keep_files = ['to_keep/child_dir']
+      @site.keep_files = ["to_keep/child_dir"]
 
       @cleaner = Cleaner.new(@site)
       @cleaner.cleanup!
     end
 
     teardown do
-      FileUtils.rm_rf(dest_dir('to_keep'))
+      FileUtils.rm_rf(dest_dir("to_keep"))
     end
 
     should "keep the parent directory" do
-      assert_exist dest_dir('to_keep')
+      assert_exist dest_dir("to_keep")
     end
 
     should "keep the child directory" do
-      assert_exist dest_dir('to_keep', 'child_dir')
+      assert_exist dest_dir("to_keep", "child_dir")
     end
 
     should "keep the file in the directory in keep_files" do
-      assert_exist dest_dir('to_keep', 'child_dir', 'index.html')
+      assert_exist dest_dir("to_keep", "child_dir", "index.html")
     end
 
     should "delete the file in the directory not in keep_files" do
-      refute_exist dest_dir('to_keep', 'index.html')
+      refute_exist dest_dir("to_keep", "index.html")
     end
   end
 
-  context "not-nested directory in keep_files and similary named directory not in keep_files" do
+  context "not-nested directory in keep_files and similary named directory not "\
+          "in keep_files" do
```

Comment:
let's try to just shorten this name. super confusing haha. how about

`non-nested directory & similarly-named directory *not* in keep_files`

does that work here?